### ISSUE TITLE
change font to character names

### ISSUE THREAD

**Issue by smoothingplane** [League-of-Foundry-Developers/fvtt-module-theatre#17]
Hi!
is there a possibility to change the font of the characters' names?

**Comment by megahead11** [comment]
Yes, I am looking to see if there is a clean way to do this however. At the moment, this requires you to go into theatre.js

You can manually do this yourself by doing two things:

At line 6142 you can edit 
`			// pre-split measurement
			insert.label.style.fontSize = 44; 
			insert.label.style.lineHeight = 44*1.5; `

This will change the font size.

Next, you can find the language of your choice  at 6498 beyond. Find `Theatre.instance.titleFont" and change it to the font of your choice in both default in and your language (in EN, it'd be case: "en")

In the near future, I will allow this to be set via an option... once I can figure out how to do so.

**Comment by IanBacklund** [comment]
Second this. I am using it for Alien and it would be nice to take some of the cartoon-ish-ness out.

**Comment by smoothingplane** [comment]
I use The Forge, I changed the files on the pc and then I uploaded them to the forge, but the "textFont" doesn't change (while on foundryvtt it works on my pc).
Furthermore it is possible that the forge will automatically update "Theater Insert" in the future by deleting the customization of the module, they recommend on their forum to change the name and title in "module.json", but doing so the module stops working.
Some advice? besides not using the forge XD
